src/utils/TicketStatusFetcher.php

```diff
@@ -6,4 +6,7 @@ interface TicketStatusFetcher
 {
     /** @return string|null Status name or null if ticket doesn't exist */
     public function fetchTicketStatus(string $ticketKey): ?string;
+
+    /** @return non-empty-string */
+    public static function getKeyPattern(): string;
```

Comment:
we should mark this interface with `@internal`  so noone outside this packages tries to implement it on its own and wonders about breaking changes in minor releases